test/fuzz/gen.ml

```diff
@@ -298,33 +381,46 @@ let global env =
 
 let local = B.param
 
-let func env =
+let func env fuzz_conf =
+  let open Crowbar.Syntax in
   let* () = const () in
   Env.reset_locals env;
   Env.refill_fuel env;
   let* locals = list (local env) in
   let* type_f = B.block_type env in
   let (_name : string) = Env.add_block env type_f Env.Func in
-  let+ body = expr ~block_type:type_f ~stack:[] ~locals env in
+  let+ body =
+    Option.get @@ run (expr ~block_type:type_f ~stack:[] ~locals env) fuzz_conf
```

Comment:
Could you explicit a little bit why `Option.get` is fine here ? By simply matching on it + a proper error message for instance.